src/components/AccountCard.tsx

```diff
@@ -80,19 +81,18 @@ export const AccountCard = ({
           asset.icons ? leftMultipleIconBoxSize : leftIconBoxSize
         } - var(--chakra-sizes-${rightIconBoxSize}))`}
       >
-        <Tooltip label={asset.name} isDisabled={!willOverflow}>
+        <Tooltip label={buildAssetTitle(asset, translate)} isDisabled={!willOverflow}>
```

Comment:
q @NeOMakinG: do we need to call a runtime method everywhere we have asset names? Looks like we would easily forget to do so.
Couldn't we make the `Asset` in store `Asset & {nameWithNetwork: string}` in the store and have this called just once and be present across the board?